swtbahn-full/displayIpAddress.py

```diff
@@ -33,8 +51,16 @@ def updateDisplay():
 		networkInterfaceIndex = 0
 
 	if len(ipCollection) > 0:
+		host_ip = None
 		host_name = ipCollection[networkInterfaceIndex][1]
-		host_ip = str(ipCollection[networkInterfaceIndex][0])
+		if networkInterfaceIndex == "wlan0":
```

Comment:
Isn't `networkInterfaceIndex` a number?

Reply:
no, its a string with the name of the network interface name